packages/client/hmi-client/src/components/workflow/ops/calibrate-ensemble-ciemss/tera-calibrate-ensemble-ciemss-drilldown.vue

```diff
@@ -205,13 +206,16 @@
 						</AccordionTab>
 						<template v-if="!isRunInProgress">
 							<AccordionTab header="Ensemble variables over time">
-								<template v-for="setting of selectedEnsembleVariableSettings" :key="setting.id">
+								<div ref="outputPanel"></div>
```

Comment:
You have removed the reference to this (ln 534) 
can this div be removed or is there something I am missing?

Reply:
I'll make the change on a separate branch that I'm currently working on